### PR TITLE
Fix #57 Add a websocket remote admin

### LINKED ISSUE #57: Allow pypeman remote shell command
This way a user can manage pypeman channels remotely on a working instance. This shell can works two ways: 
- Ipython API shell
- Specific shell

We should be able to: 
- List available channels
- List messages from a channel message store
- Replay a message from a message store
- Inject new message

### PR DESCRIPTION
**Pull request by jrmi**
You now can use option `--remote-admin` to enable listening websocket to send remote command to running instance of pypeman.

This allow you to start and stop channels, inject new message or replay failed message from message store.

This work is not finished yet but don't break things so can be  integrated as is.

Main modifications are:
- Enhance documentation
- Fix some bug in message with uuid
- Add remoteadmin module
- Add two commands

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -7,48 +7,55 @@
...
+        'Topic :: Internet :: File Transfer Protocol (FTP)',
+        'Programming Language :: Python',
+    ],
+    keywords='esb etl eai pipeline data processing asyncio http ftp hl7',
+    url=URL,
 
-      author='Jeremie Pardou',
-      author_email='jeremie.pardou@mhcomm.fr',
+    author='Jeremie Pardou',
+    author_email='jeremie.pardou@mhcomm.fr',
```
Comment (Zluurk) [inline setup.py]: maibe change your email ...

File: pypeman/remoteadmin.py [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+        chan = self.get_channel(channel)
+        await chan.start()
+
+    async def list_msg(self, channel):
+        """
+        List first 10 messages from message store of specified channel.
+
+        :params channel: The channel name.
+        """
+        # TODO allow indexing
```
Comment (Zluurk) [inline pypeman/remoteadmin.py]: why just 10 ? param maybe

File: setup.py (hunk 1 above)
Comment (jrmi) [reply setup.py]: Ok, next PR, this one is yet valid ;-)

File: pypeman/remoteadmin.py (hunk 2 above)
Comment (jrmi) [reply pypeman/remoteadmin.py]: Yes, I want to do it in a next PR as I have to modify message store also...

**Comment by Zluurk** [comment]
ok for me

**Pull request merged**